packages/blade/src/components/ActionList/ActionListItem.tsx

```diff
@@ -257,14 +257,12 @@ const ActionListItem: WithComponentId<ActionListItemProps> = (props): JSX.Elemen
           props.onClick?.({ name: props.value, value: isSelected });
         })}
         {...metaAttribute(MetaConstants.Component, MetaConstants.ActionListItem)}
-        onFocus={() => {
-          // We don't want to keep the browser's focus on option item. We move it to selectInput
-          if (!isReactNative()) {
-            triggererRef.current?.focus();
-          }
-        }}
         onMouseDown={() => {
           setShouldIgnoreBlur(true);
+          setShouldIgnoreBlurAnimation(true);
+        }}
+        onMouseUp={() => {
+          setShouldIgnoreBlurAnimation(false);
```

Comment:
We can add some comments on why to ignore blur animation on mouse down but not on mouse up

Reply:
done